field/TinyLang/Field/Rename.hs

```diff
@@ -8,9 +9,14 @@ import           TinyLang.Field.Typed.Core
 
 renameExpr :: MonadSupply m => Expr f a -> m (Expr f a)
 renameExpr expr = do
```

Comment:
Let's remove that for now, we're mostly interested in renaming programs rather than standalone expressions.

Reply:
Done